mvpa2/tests/test_shrinklda.py

```diff
@@ -0,0 +1,74 @@
+# emacs: -*- mode: python; py-indent-offset: 4; indent-tabs-mode: nil -*-
+# vi: set ft=python sts=4 ts=4 sw=4 et:
+### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ##
+#
+#   See COPYING file distributed along with the PyMVPA package for the
+#   copyright and license terms.
+#
+### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ##
+"""Unit tests for PyMVPA Shrinkage LDA classifier"""
+# based on test_gnb.py
+
+import numpy as np
+
+from mvpa2.clfs.skl.base import SKLLearnerAdapter
+
+from mvpa2.misc.data_generators import normal_feature_dataset
+from mvpa2.measures.base import CrossValidation
+from mvpa2.generators.partition import NFoldPartitioner
+from mvpa2.testing import *
+from mvpa2 import externals
+
+class ShrinkageLDATests(unittest.TestCase):
+
+    def test_shrinkage_lda_faster_than_skl(self):
+        # Compare PyMVPA's shrinkage LDA implementation with scikit-learn's
+        # implementation when used through SKLLearnerAdapter. Are prediction
+        # accuracies next to identical? Is the PyMVPA implementation faster?
+
+        # First check if scikit-learn is available at all
+        if not externals.exists('skl') or externals.versions['skl'] < '0.17':
+            self.skipTest('sklearn (0.17 or higher) unavailable')
```

Comment:
ha -- never used it this way ... according to docs this should indeed raise an exception. So no need for explicit `else:` block indentation -- could be dedented since would not run if there is old sklearn and this exception is raised.

anyways -- you could just use a line
```
skip_if_no_external('skl', min_version='0.17')